src/script/components/MessagesList/Message/ContentMessage/asset/MultipartAssets/MultipartAssets.tsx

```diff
@@ -84,7 +84,7 @@ const MultipartAsset = ({
   senderName,
   timestamp,
 }: MultipartAssetProps) => {
-  const name = trimFileExtension(initialName!);
+  const name = getName(initialName!);
```

Comment:
how about we make getName accept nullable parameters and remove this ! from here?

Reply:
I already use `!` in other places; I'd like to stick with this convention. It's shouldn't be optional, we'll have to change it in the protocol